column.go

```diff
@@ -707,7 +707,7 @@ func decodeLevelsV2(enc encoding.Encoding, numValues int, data []byte, length in
 }
 
 func decodeLevels(enc encoding.Encoding, numValues int, data []byte) (levels *buffer, err error) {
-	levels = levelsBufferPool.get()
+	levels = levelsBufferPool.get(len(data) * 8)
```

Comment:
It seems like maybe this might be a bit too conservative? Nested columns would result in repetition or definition levels that are encoded with more than one bit, would it make this call to `get` always allocate a new buffer then?

Reply:
yeah, i presume that the next line `enc.DecodeLevels` would have to resize the buffer accordingly. any guesses on a constant that makes more sense? 16? 32?

Reply:
ok, i feel silly. The parameter `numValues` tells us exactly how big the buffer needs to be. fixed.